src/main/java/com/example/petbuddybackend/service/session/chat/WebsocketChatMessageSender.java

```diff
@@ -0,0 +1,141 @@
+package com.example.petbuddybackend.service.session.chat;
```

Comment:
dalbym to normalnie do service.chat, bo tutaj nie zarzadzamy sesja, tylko wysylamy wiadomosci